packages/typespec-ts/src/modular/buildLroFiles.ts

```diff
@@ -376,10 +358,6 @@ export function buildGetPollerHelper(
           options.abortSignal?.removeEventListener("abort", abortListener);
           pollOptions?.abortSignal?.removeEventListener("abort", abortListener);
         }
-        if (options.initialRequestUrl || initialResponse) {
-          response.headers["x-ms-original-url"] =
-            options.initialRequestUrl ?? initialResponse!.request.url;
-        }
```

Comment:
why this is removed?

Reply:
The reason this header value was being set seems to be because the `isUnexpected` helper was using it. Since we're not using `isUnexpected` anymore I think it's ok to remove. Unless there's another place where we're using this value I don't know about?